linera-storage/src/db_storage.rs

```diff
@@ -1157,3 +1163,168 @@ where
         Ok(Self::new(database, wasm_runtime, clock))
     }
 }
+
+#[derive(Default, Debug, Eq, PartialEq, Serialize, Deserialize)]
+#[repr(u8)]
+enum SchemaDescription {
+    /// Version 0, all the blobs, certificates, confirmed blocks, events and network description on the same partition.
+    /// This is marked as default since it does not exist in the old scheme and would be obtained from unwrap_or_default
+    #[default]
+    Version0SingleBlobPartition,
+    /// Version 1, spreading by ChainId, CryptoHash, and BlobId.
+    Version1MultiBlobPartition,
+}
+
+/// The key containing the schema of the storage.
+const SCHEMA_ROOT_KEY: &[u8] = &[0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233];
+
+#[derive(Debug, Serialize, Deserialize)]
+enum BaseKey {
```

Comment:
I assume this is the V0 BaseKey.
Can you define a module `Version0` and move the V0 definitions into this module for namespacing purpose?

Reply:
Alternatively (better?), we could have a file `db_storage_v0.rs`